website/content/docs/security/acl/auth-methods/kubernetes.mdx

```diff
@@ -158,3 +158,20 @@ use in binding rule selectors and bind name interpolation.
 | `serviceaccount.namespace` | Equal, Not Equal, In, Not In, Matches, Not Matches | yes                 |
 | `serviceaccount.name`      | Equal, Not Equal, In, Not In, Matches, Not Matches | yes                 |
 | `serviceaccount.uid`       | Equal, Not Equal, In, Not In, Matches, Not Matches | yes                 |
+
+## Token Locality
+
+By default, tokens created by the Kubernetes Auth Method will have local scope
+and will therefore only be valid for the datacenter in which they were created.
+On the other hand, by using [`-token-locality=global`](/commands/acl/auth-method/create#token-locality),
+tokens created by the Auth Method will be replicated to all other datacenters.
+What this means is that by using `global` locality, there are implications on
+the Consul servers' access to associated Kubernetes clusters. This may come as
+a limitation to the given Consul deployment so using
```

Comment:
Based on our discussion yesterday, I think we might need to adjust this a bit. Maybe something about along the lines of:
*  generally the kubernetes auth method is only intended to create tokens for services, not for humans
* each k8s auth method can only validate JWT tokens from a single k8s cluster
* since an auth method with token-locality=global can only be created in the primary DC, the primary DC would  need a k8s auth method for each k8s cluster in every dc , and Consul servers in  the primary DC would need direct network access to each of them
* this is often an undesirable setup, so configuring a policy on the anonymous token may be an easier way to allow cross-DC requests in kuberentes

Probably with a call out before all of this as Luke mentioned to say that anyone using consul-k8s with managedACLs does not need to care about this because it's handled for them.

Not sure if this helps, you probably already have better notes on this.